src/main/scala/org/ergoplatform/local/TransactionGenerator.scala

```diff
@@ -46,10 +46,8 @@ class TransactionGenerator(viewHolder: ActorRef, settings: TestingSettings) exte
         }
       }*/
 
-    case txs: Seq[ErgoTransaction] =>
-      txs.foreach { tx =>
+    case tx: ErgoTransaction =>
```

Comment:
What is the reason to make this change for the class which is not working (and not sending to self neither an instance of ErgoTransaction nor Seq[ErgoTransaction] ?

Reply:
This string generated a warning due to "type pattern is unchecked since it is eliminated by erasure" and required to be fixed. I may just remove this string, it's hard to fix correctly, because class is not implemented